src/plugins/home/public/application/components/solutions_section/solution_panel.tsx

```diff
@@ -64,9 +64,10 @@ interface Props {
   addBasePath: (path: string) => string;
   solution: FeatureCatalogueSolution;
   apps?: FeatureCatalogueEntry[];
+  branding: Branding;
 }
 
-export const SolutionPanel: FC<Props> = ({ addBasePath, solution, apps = [] }) => (
+export const SolutionPanel: FC<Props> = ({ addBasePath, solution, apps = [], branding }) => (
```

Comment:
NIT: since we are did we want to take the time add some jsdoc here. Just improve the coding standards of the repo in general is win for me.

Reply:
Jsdoc are added for every functions within solution_panel.tsx file.